Display sponsors by contribution tier

will-hou opened the pull request:
Currently, all sponsors are displayed in alphabetical order.

This PR displays sponsor logos under a banner corresponding to their sponsor tier. Additionally, it re-organizes the navbar so that the Sponsors page can be accessed directly (instead of under the Community tab)

will-hou commented:
Also, this is my first time doing any real work on the website so please let me know if there are things that I should've done differently to follow code/style guidelines :)

andrewda commented on assets/css/pages.css:
```diff
@@ -32,14 +32,30 @@
 }
 
+.sponsor-tier-banner {
+  height: 15vh;
```
Could this be changed to 10vh? I think it looks better a tad bit smaller.

andrewda commented on assets/css/pages.css:
```diff
@@ -32,14 +32,30 @@
+  height: 15vh;
+  background: #2b2046ff;
+  width: 100%;
+  margin-bottom: 10vh;
```
Also if this could be changed to 5vh -- it helps visually combine the header and logos.

andrewda commented on _data/navigation.yml:
```diff
@@ -20,5 +20,6 @@
-  - url: /sponsors
-    title: Sponsors
+
+- url: /sponsors
```
keep formatting as it was

will-hou replied:
I talked with Meredith a little before hand and we felt that it'd be good to have our sponsors page be a little more visible since it'll still fit outside of a drop-down menu. I'm probably not aware of the reason for lumping it in a drop down with Community though. 

What would you think of eliminating the drop-down all together and just give Outreach and Sponsors their own place on the navbar?

andrewda replied:
Oh whoops, I didn't actually realize that's what this was doing, my bad! The only issue I see is that, on mobile, it makes the navbar 2 lines (we need to redesign the mobile site anyways so not a _huge_ deal). How would you feel about moving "Parents" to the community tab?

will-hou replied:
Good idea!

will-hou replied:
Actually, on second thought, how does moving "Leadership" under "About" sound? And then we can just replace "Community" with "Outreach." Parents doesn't seem to quite fit under 'Community' to me

andrewda replied:
That works for me, but move all the dropdowns to the end of the row. I think robots might also fit into that drop-down.

will-hou replied:
👍 I kept the "About" button where it was before for now -- I feel like most people will assume that it's a dropdown. I feel like "Robots" is a rather important page and should be visible at the top-level so I still put "Parents" under "Community" and moved it to the end so that we wouldn't have too many things on the navbar

will-hou commented:
Just realized that it's very difficult to click on the "Leadership" button on mobile because tapping "About" automatically takes the user to that page before they can click on any of the options in the drop down.

As a temporary fix, I removed the link from the "About" button and added a "Who We Are" button on the drop-down menu linking to /about instead. Not being able to click on "About" directly feels really icky but I hope it'll do for now until we make some changes to navigation on mobile

will-hou commented:
Thanks for taking the time to look at it :)
